main.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/elazarl/goproxy"
 	yaml "gopkg.in/yaml.v2"
 
-	"github.com/mrkschan/cuttle/cuttle"
+	"./cuttle"
```

Comment:
Needed this to build properly locally, but it can be reverted prior to merging if it works better as a full url like this.